package.json

```diff
@@ -111,7 +111,8 @@
         "wdio-spec-reporter": "^0.1.5",
         "webdriverio": "^4.14.1",
         "node-red-node-test-helper": "node-red/node-red-node-test-helper",
-        "jsdoc-nr-template": "node-red/jsdoc-nr-template"
+        "jsdoc-nr-template": "node-red/jsdoc-nr-template",
+        "iconv-lite": "^0.4.24"
```

Comment:
This needs to be moved under the main `dependencies` section - we also use fixed version numbers for main dependencies.

It also needs adding into the package.json file of the @node-red/nodes modules - https://github.com/node-red/node-red/blob/dev/packages/node_modules/%40node-red/nodes/package.json#L17.

Reply:
Fixed package.json